__tests__/integration.test.js

```diff
@@ -50,7 +50,7 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/1")
       .expect(200)
       .then(({ body: { article } }) => {
-        expect(Object.keys(article)).toHaveLength(8);
+        expect(Object.keys(article)).toHaveLength(9);
```

Comment:
the problem with using object.keys here is it makes the test brittle like here it now needs to be changed to pass! I wouldnt use it personally!

Reply:
removed